test/spec/runners/epoch_processing.ex

```diff
@@ -14,7 +14,6 @@ defmodule EpochProcessingTestRunner do
     "registry_updates",
     "slashings",
     "eth1_data_reset",
-    "effective_balance_updates",
```

Comment:
Apart from commenting this out, you should create an actual tests that assert that 

`process_effective_balance_updates(pre) == post`